Makefile

```diff
@@ -18,5 +22,8 @@ check-go-test:
 check-go-static:
 	.ci/go-static-checks.sh $(GO_STATIC_CHECKS_ARGS)
 
+install:
+	$(QUIET_INST)install -D $(TARGET) $(DESTDIR)$(LIBEXECDIR)/clearcontainers/$(TARGET) || exit 1;
```

Comment:
Why is this being installed in libexec dir? The runtime can be called standalone, so it would be make sense to install this under /usr/bin so that it is found in the search path. Also, this makes it possible to add the runtime to docker without specifying the complete path.